src/App.js

```diff
@@ -1,11 +1,39 @@
-import "./App.css";
-import PeriodicTable from "./Components/PeriodicTable.js";
+import PeriodicTable from './Components/PeriodicTable.js';
+import styled from 'styled-components';
 
 export default function App() {
   return (
-    <div className="App">
-<h1>Periodic Table of Elements</h1>
-<PeriodicTable />
-    </div>
+    <StyledBody>
+      <header>
+        <h1>Periodic Table of Elements</h1>
+      </header>
+      <main>
+        <PeriodicTable />
+      </main>
+
+      {/* <nav>
+          <p>navigation test</p>
+          <p>navigation test</p>
+          <p>navigation test</p>
+        </nav> */}
+    </StyledBody>
   );
 }
+
+const StyledBody = styled.div`
+  overflow-y: scroll;
+  overflow-x: scroll;
+  scrollbar-width: none;
+`;
+
+const MyHeader = styled.header`
+  h1 {
+    color: red;
+  }
+`;
```

Comment:
You have a code quality issue here as you do not use this styled component. As suggested above, I would get rid of the header and style your heading directly, e.g.:
```suggestion
const Heading = styled.h1`
    color: red;
`;
```